arcgis-ios-sdk-samples/Maps/Generate offline map (basemap by reference)/README.metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "category": "Maps",
+    "description": "@@TODO: Use the AGSOfflineMapTask to take a web map offline, but instead of downloading an online basemap, use one which is already on the device.",
+    "ignore": false,
+    "images": [
+        "image1.png"
+    ],
+    "keywords": [
+        "Offline",
+        "AGSOfflineMapTask",
```

Comment:
It looks like the relevant APIs are being appended to the keywords. Might it be useful to keep those separate? They could be combined later at the point of use, correct?

Reply:
Yes, this is by design -- the keywords is used by page SEO, so I think we want to the class names in the page metadata so the sample pages will have a higher ranking on google, etc.